homeworks/karolinavalko/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,23 @@
+name = input("What is your name? ")
+age = int(input("What is your age? "))
+experience_in_Python = int(
+    input("How many years of experience do you have in Python? ")
+)
+developper_desire = (
+    input("Would you like to be a professional Python developper? Yes or no? ") == "yes"
+)
+
+
+name = name.strip().title()
```

Comment:
Szuper a method chaining használat! Nem hiba, de ha akarod akkor ezt akár átviheted az első sorra is:
name = input("What is your name? ").strip().title(), így egy soron lerendezel minden névvel kapcsolatos teendőt.